src/views/token-swap/components/quote/quote.styles.js

```diff
@@ -72,22 +72,6 @@ const useQuoteStyles = createUseStyles((theme) => ({
     color: theme.palette.red.main,
     marginTop: theme.spacing(1),
   },
-  rewardCell: {
-    flex: "1 1 20%",
-    display: "flex",
-    alignItems: "center",
-    justifyContent: "center",
-    flexDirection: "column",
-  },
-  reward: {
```

Comment:
Token swap use different rewards. They are not related to this issue

Reply:
Thx! Reverted!